mlir/test/Dialect/LLVMIR/genx.mlir

```diff
@@ -34,3 +34,9 @@ func.func @genx.barrier() {
   genx.barrier
   llvm.return
 }
+
+llvm.func @genx.atomic.cmpxchg.i32(%ptr : !llvm.ptr<i32>, %cmp : i32, %val : i32) -> i32 {
+  // CHECK: genx.atomic.cmpxchg %arg0, %arg1, %arg2 : (!llvm.ptr<i32>, i32, i32) -> i32
```

Comment:
```suggestion
  // CHECK: %0 = genx.atomic.cmpxchg %arg0, %arg1, %arg2 : (!llvm.ptr<i32>, i32, i32) -> i32
```

Reply:
Because the return value is not used I did not put it in. The function must return a value because the function type `(!llvm.ptr<i32>, i32, i32) -> i32` requires it.

Reply:
ok